src/main/java/frc/robot/turret/simulation/SimulationTurret.java

```diff
@@ -12,13 +13,15 @@
 
 import static frc.robot.turret.simulation.SimulationConstants.GEARING;
 import static frc.robot.turret.simulation.SimulationConstants.JKG_METER_SQ;
+import static frc.robot.turret.simulation.SimulationConstants.MOTOR_CONFIGURATION;
 
 
 public class SimulationTurret implements ITurret {
 
     private SimpleMotorSimulation motor;
     private VelocityDutyCycle velocityDutyCycle;
     private PositionDutyCycle positionDutyCycle;
+    private VoltageOut voltageOut;
 
     public SimulationTurret() {
         motor = new SimpleMotorSimulation(
```

Comment:
this.motor

Reply:
out of scope but i'll give you that. you only need to comment about what's in the scope of the PR and not the whole code.